### ISSUE TITLE
GRPC endpoints for Paxos commit protocol

### ISSUE THREAD

**Issue by g4s8** [cqfn/degitx#236]
Let's add GRPC services for acceptors and proposers, let's put it to the same package as origin types (base package will be`cqfn.org/degitx/pkg/paxos/`). Keep in mind that each transaction serves multiple Paxos instances simultaneously, and it's possible to run multiple transactions simultaneously too, but each node serves only one GRPC service for all Paxos instances. So we need to pass a context about transaction and Paxos instance (maybe using metadata headers/context), and redirect requests/responses to correct instance from GRPC service.

Transaction participants: each RM (resource manager) node hosts one Proposer, and n Acceptors, where n is a number of RM, also TM (transaction manager) hosts n Acceptors but no Proposers. Each Acceptor is attached to RM, so we may use RM id for request context.

Also, each RM and TM may server multiple transactions, where each transaction will have `n` Paxos instances, so we may use transaction id to distinguish transactions.

In other words, a combination of transaction id and RM id would be necessary and sufficient to identify Paxos instance in a multi-transactional model.

So transaction for Acceptors mapping in RM GRPC server is a many-to-many relation of transaction id to Paxos instance, and Paxos instance is mapping with many-to-many relation of RM id to Acceptor.
